test/browser/named-register.js

```diff
@@ -11,7 +11,7 @@ suite('Named System.register', function() {
 
   test('Loading a named AMD bundle', function () {
     return System.import('./fixtures/browser/named-amd.js').then(function (m) {
-      assert.equal(Object.keys(m).length, 2);
+      assert.equal(m.a, 'b');
```

Comment:
This test was making a weird assertion that I think was wrong. On the master branch, `m` is `{__useDefault: true, default: {}}`. Even though `named-amd.js` registers two modules that have values in them.

With my changes in this PR, `m` becomes the `c` module [defined here](https://github.com/systemjs/systemjs/blob/master/test/fixtures/browser/named-amd.js#L1). This is similar behavior to how the named-register extra [keeps track of the firstNamedDefine](https://github.com/systemjs/systemjs/blob/master/src/extras/named-register.js#L20). I updated the assertions to verify that the `c` module is being returned now.

Reply:
This seems ok to me.